system/kube-monitoring/charts/prometheus-frontend/openstack-swift.alerts

```diff
@@ -0,0 +1,213 @@
+    dashboard = "swift-overview"
+  }
+  ANNOTATIONS {
+    summary = "swift-container-first-byte-timing",
+    description = "Alert to notify if the swift container first-byte-timing reached more than 500 within 3 minutes.",
+  }
+  
+ALERT swift-object-first-byte-timing
+  IF max(swift_proxy_firstbyte_timer{status="200",type="object",quantile="0.5"}) > 500.0
+  FOR 3m
+  LABELS {
+    service = "swift"
+    severity = "critical"
+    context = "openstack"
+    dashboard = "swift-overview"
+  }
+  ANNOTATIONS {
+    summary = "swift-object-first-byte-timing",
+    description = "Alert to notify if the swift object first-byte-timing reached more than 500 within 3 minutes.",
+  }  
+  
+ALERT swift-health-check
+  IF avg(swift_health_statsd_exit_code_gauge{}) > 0.2
+  FOR 5m
+  LABELS {
+    service = "swift"
+    severity = "critical"
+    context = "openstack"
+    dashboard = "swift-overview"
+  }
+  ANNOTATIONS {
+    summary = "swift-health-check",
+    description = "Swift health check failures. Run kubectl log swift-proxy-... collector to get details",
+  }    
+
+ALERT swift-used-space-75
+  IF max(swift_cluster_storage_used_percent_gauge{}) > 0.75
+  FOR 3m
+  LABELS {
+    service = "swift"
+    severity = "critical"
+    context = "openstack"
+    dashboard = "swift-overview"
+  }
+  ANNOTATIONS {
+    summary = "swift-used-space-75",
+    description = "Swift storage usage reached 75%",
+  } 
+
+ALERT swift-used-space-70
+  IF max(swift.cluster.storage_used_percent,180) > 0.7
+  FOR 3m
+  LABELS {
+    service = "swift"
+    severity = "warning"
+    context = "openstack"
+    dashboard = "swift-overview"
+  }
+  
+  ANNOTATIONS {
+    summary = "swift-used-space-70",
+    description = "Swift storage usage reached 70%",
+  }
+  
+ALERT swift-overlapping-objects
+  IF min(swift.dispersion.object_overlapping{}, 180) > 0
+  FOR 3m
+  LABELS {
+    service = "swift"
+    severity = "warning"
+    context = "openstack"
+    dashboard = "swift-overview"
+  }
+  ANNOTATIONS {
+    summary = "swift-overlapping-objects",
+    description = "Swift has overlapping objects. Run swift-dispersion-report to get details",
+  }
+  
+ALERT swift-overlapping-containers
+  IF MIN(swift.dispersion.container_overlapping{}, 180) > 0.0
+  FOR 3m
+  LABELS {
+    service = "swift"
+    severity = "warning"
+    context = "openstack"
+    dashboard = "swift-overview"
+  }
+  ANNOTATIONS {
+    summary = "swift-overlapping-containers",
+    description = "Swift has overlapping containers. Run swift-dispersion-report to get details",
+  } 
+
+ALERT swift-ntpoffset
+  IF MIN(ntp.offset{service=object-store}, 180) < -0.3 or MAX(ntp.offset{service=object-store}, 180) > 0.3
+  FOR 3m
+  LABELS {
+    service = "swift"
+    severity = "warning"
+    context = "openstack"
+    dashboard = "swift-overview"
+  }
+  
+  ANNOTATIONS {
+    summary = "swift-ntpoffset",
+    description = "Swift NTP offset is over +/- 300ms on {{systemtype}} {{hostname}}",
```

Comment:
Forward the labels (`by (systemtype)`) and access them here via `{{ $labels.systemtype }}` and `{{ $labels.hostname }}`.